cmd/crc/cmd/setup.go

```diff
@@ -31,6 +32,20 @@ var setupCmd = &cobra.Command{
 }
 
 func runSetup(arguments []string) error {
+	if config.Get(cmdConfig.ConsentTelemetry).AsString() == "" {
```

Comment:
Now we are not adding the force option, CI will fail because it need the input?

Reply:
Yes this is what I want to try. `input.PromptUserForYesOrNo` checks if the sessions is interactive. https://ci.centos.org/job/codeready-crc-pr/2754/console seems to pass so it correctly falls in the no.

Reply:
That's nice, I didn't know that was a thing... So I could theoretically change `delete -f` instances to `delete` in the tests?